lib/core/src/lib/card-view/components/card-view-boolitem/card-view-boolitem.component.ts

```diff
@@ -22,7 +22,14 @@ import { BaseCardView } from '../base-card-view';
 
 @Component({
     selector: 'adf-card-view-boolitem',
-    templateUrl: './card-view-boolitem.component.html'
+    templateUrl: './card-view-boolitem.component.html',
+    styles: [
```

Comment:
Isn't it better and more clear to move it to .scss file?

Reply:
The only reason I added it here is because it's only a single class, and the scss file did not exist. However, if needed, I can create a new file and add the styling to that instead.